handlers_test.go

```diff
@@ -9,46 +9,8 @@ import (
 	"testing"
 )
 
-func Test_GivenNonExistingRoute_RouterReturns404(t *testing.T) {
```

Comment:
I thought you will leave those tests here for the reference in the future since `httptest` is a neat thing; but if you like them gone I'm fine with that :smile:

Reply:
Oh you have just moved them.. nvm then :stuck_out_tongue: